README.md

```diff
@@ -1,5 +1,9 @@
 # test-fuzz
 
+**News:** `cargo-test-fuzz` 5.3.0 enables [`AFL_SHA1_FILENAMES`] by default. If this causes you problems, please [open an issue], and reinstall `cargo-test-fuzz` with `--no-default-features`.
```

Comment:
This is the wrong approach. Once [this fix](https://github.com/AFLplusplus/AFLplusplus/pull/2202) is published, we can start testing in CI. Then, after some time has passed, we can enable [`AFL_SHA1_FILENAMES`](https://github.com/AFLplusplus/AFLplusplus/blob/stable/docs/env_variables.md) with a major version bump.